bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -6509,12 +7205,20 @@ msgid "Validation failed: '${keyword}': duplicate keyword"
 msgstr "La validación ha fallado: '${keyword}': palabra clave duplicada"
 
 #: bika/lims/validators.py:274
-msgid "Validation failed: '${title}': This keyword is already in use by calculation '${used_by}'"
-msgstr "La validación ha fallado: '${title}': la palabra clave ya está en uso en el cálculo '${used_by}'"
+msgid ""
+"Validation failed: '${title}': This keyword is already in use by calculation "
+"'${used_by}'"
+msgstr ""
+"La validación ha fallado: '${title}': la palabra clave ya está en uso en el "
```

Comment:
@xispa we should change _La validación ha fallado_ for _Error de validación_ to be coherent